### DISCUSSION TITLE
Using ExPutt alone

### DISCUSSION THREAD

**Discussion by hgspch** [springbok/MLM2PRO-GSPro-Connector#79] [Q&A] [answered]
Hi, 

Is it possible to use ExPutt as the only connector, so nothing is set in the Connector > Launch Monitor section ?

Everything seems correctly set  up for ExPutt, but no ball data get displayed in the Springbok connector, and nothing happens in GsPro.

I need to have just ExPutt in the Springbok connector, because my launch monitor is a Garmin R10, so using another connector. And being able to use only the Exputt, to practice putting, would be cool.

![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/74004/b8a62101-dac7-4946-ac49-8a15608f46d3)
![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/74004/79b68534-2400-4e3b-991d-948a874f5a18)
![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/74004/fd5122ed-6a1c-4b1b-b8d4-ad017800b9ff)
![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/74004/0d43afd3-78e6-4ea0-88c7-2bde84c00fcd)

**Comment by hgspch** [comment]
Note : in GsPro, "Putter" is selected, and "Putting Mode" is selected.

**Comment by springbok** [comment]
You can’t have two connections to gspro. I’m currently working on using the R10 connector which will connect to the mlm connector then to gspro allowing you to use exputt and still have a direct connect to r10.

**Reply by hgspch** [reply]
Good news for the R10 !
But can we just use the ExPutt alone, for putting practice ?

**Reply by hgspch** [reply]
Note : screenshots above are with ExPutt only, no R10 involved.

**Reply by hgspch** [reply]
Actually, the "Putter" label does not show up in the GSPro Connection panel :
![image](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/74004/84c9b8cc-8aa1-4bbd-b05a-05f46f48e911)

**Comment by hgspch** [comment, marked as answer]
Solved with issue https://github.com/springbok/MLM2PRO-GSPro-Connector/issues/81 👏

**Comment by springbok** [comment]
FYI, I've added a Garmin R10 Bluetooth connection to the connector if you want to give it a test run.

https://1drv.ms/u/c/7c92ade387697b0c/ERzsMDn8kDJApVB2aiGYjiUBLSlJpjMy8HI__N924wGsCw?e=AmdRMm

**Reply by hgspch** [reply]
Thanks for the information. 
Actually, I switched to Mlm2pro. Much better learning tool with the videos, and accurate shot shapes.

**Reply by springbok** [reply]
I’ve also added in a MLM2PRO Bluetooth connector :)

**Reply by hgspch** [reply]
Oh, cool.
But you have to use the "3rd party app" mode on the Rapsodo app, so no more access to impact vision ?
Any chance to have both ?
